src/main/java/slimeknights/tconstruct/library/tools/capability/TinkerDataKeys.java

```diff
@@ -42,6 +43,9 @@ static void init() {}
   /** Soul belt level for hotbar preservation */
   TinkerDataKey<Integer> SOUL_BELT = intKey("soul_belt");
 
+  TinkerDataKey<Integer> MELEE_PROTECTION = intKey("melee_protection");
```

Comment:
As mentioned above, we don't need this. The reason for the key registry is because we are going to make use of these keys in Java code. If the key is only being used inside a single module, it need not be part of the registry; just create it in the module in a non-static field.